db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
```

Comment:
I'll ignore this for now, but please use `git commit -p` or similar in the future to avoid mass-changing of whitespace in the schema files.